lib/Dialect/TTIR/Transforms/Passes.cpp

```diff
@@ -1608,4 +1609,22 @@ class TTIRGenericOpCBs : public impl::TTIRGenericOpCBsBase<TTIRGenericOpCBs> {
   }
 };
 
+class TTIRPrivatizeFunctions
+    : public impl::TTIRPrivatizeFunctionsBase<TTIRPrivatizeFunctions> {
+public:
+  using impl::TTIRPrivatizeFunctionsBase<
+      TTIRPrivatizeFunctions>::TTIRPrivatizeFunctionsBase;
+
+  void runOnOperation() final {
+    ModuleOp module = getOperation();
+    module.walk([](mlir::func::CallOp callOp) {
```

Comment:
nit: make callOp const

Reply:
`CallOp::getCallee()` is not `const` so if I want to call it I can't mark `callOp` const.

Reply:
Oh interesting, thanks for letting me know!